### ISSUE TITLE
Missing landing page to show node status

### ISSUE THREAD

**Issue by sveip** [jascdk/Nilan_Homeassistant#14]
What I would like to add, is more info to the root web-page. As of now I cant see any way to get any sensible data from the node if the serial isn't connected (correctly) to the Nilan. It would be great to have a landing page (index.html) that shows serial connectivity status etc.

**Comment by jascdk** [comment]
@sveip hi :) have you found out something for this ?

**Comment by sveip** [comment]
Not yet, haven't had a chance to look at it yet.

**Comment by jascdk** [comment]
> Not yet, haven't had a chance to look at it yet.

Hi @sveip are you on Facebook messenger? :)

**Comment by sveip** [comment]
Sorry for the delay, been a busy week. Will try to push a PR tonight.

**Comment by jascdk** [comment]
Nice - looking forward to see what you come up with ;)

**Comment by sveip** [comment]
Got something that works ok, but not perfect. Seems to be an issue with
closing the client request, will continue tomorrow. :-)

**Comment by jascdk** [comment]
Just qurious - is it the landing page your are working on or some kind of wifimanager - integration? :)

**Comment by sveip** [comment]
No, what I implementer now is a semi-static page that just shows some
welcome text and the WIFI SSID and MQTT server details. I can push a PR as
is on a branch when I come home, so that you can test it out.

tor. 6. feb. 2020 kl. 13:44 skrev jascdk <notifications@github.com>:
